custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -129,6 +128,12 @@ def get_person_case_structure(self):
             },
         }
 
+        if self._location:
```

Comment:
sorry, when, I said "fail", I meant that we should still assign to the `_archive_` owner id, set the `archive_reason`, and `migration_error` to `migration_not_phi_location` or something.